packages/queue/drizzle/0000_blue_weapon_omega.sql

```diff
@@ -0,0 +1,18 @@
+CREATE TABLE `tasks` (
```

Comment:
if we had the bookmark id as a field, we could do queries on them, without having to look into the payload.
To my knowledge, everything done in the worker is currently always related to a bookmark?

Reply:
Currently, yes. But it doesn't have to stay like that. I already have plans to include a worker to periodically reap orphaned assets for example. So I want to keep the queues generic.

Reply:
we can have that as a nullable field then?